scipy/sparse/construct.py

```diff
@@ -61,7 +61,7 @@ def spdiags(data, diags, m, n, format=None):
     return dia_matrix((data, diags), shape=(m,n)).asformat(format)
 
 
-def diags(diagonals, offsets, shape=None, format=None, dtype=None):
+def diags(diagonals, offsets=None, shape=None, format=None, dtype=None):
```

Comment:
Just keeping in with the other default arguments. Definitely didn't want it accidentally replaced with a `[0]` accidentally later.